packages/dd-trace/src/config.js

```diff
@@ -390,6 +413,85 @@ class Config {
     }
   }
 
+  _getStableConfigPaths () {
```

Comment:
There are a lot of new helpers and new code here all related to stable config, should it be extracted maybe to a separate file that is used by `config.js`?

Reply:
cc @khanayan123 since I know you've been looking into splitting up this file.

Reply:
Split in [split into two files](https://github.com/DataDog/dd-trace-js/pull/5126/commits/d683435d48353dd4b55f4617e1d4817f8b77860f)